lambdas/config.py

```diff
@@ -51,6 +52,15 @@ def configure_logger(
     )
 
 
+# NOTE: FEATURE FLAG: function will be removed after v2 work is complete
```

Comment:
The more we can follow this comment convention of `# NOTE: FEATURE FLAG: ...` the easier I think it'll be to find feature flag branching logic in code when we're ready to remove!  Maybe/probably better ways, but hopeful this will work, thanks for implementing.